gopher/annotations.py

```diff
@@ -3,6 +3,7 @@
 import pandas as pd
 
 from . import utils, config, ontologies
+import random
```

Comment:
**suggestion:** Instead of using random here, I recommend using uuid: https://docs.python.org/3/library/uuid.html

Reply:
Fixed.